test/ActorStateUnitTests.hs

```diff
@@ -0,0 +1,23 @@
+module ActorStateUnitTests (actorStateUnitTests) where
+
+import Prelude ()
+
+import Game.LambdaHack.Core.Prelude
+
+import Test.Tasty
+import Test.Tasty.HUnit
+
+import           Game.LambdaHack.Common.ActorState
+
+import           Game.LambdaHack.Definition.Ability as Ability
+import           UnitTestHelpers
+
+actorStateUnitTests :: TestTree 
+actorStateUnitTests = testGroup "actorStateUnitTests" 
+  [ testCase "getActorBody verify stubCliState has testActor" $
+    do getActorBody testActorId (cliState stubCliState) @?= testActor
```

Comment:
this `do` and all others, are not needed when you have a single expression (in this case, the `@?=` expression), because `do` gives a special syntax for sequencing operations (by putting them in subsequent lines) and that's all it does

Reply:
(and the sequencing may use the `a <- b` syntax, additionally, binding variable a)